app/javascript/crayons/Buttons/Button.jsx

```diff
@@ -30,8 +30,8 @@ export const ButtonNew = (props) => {
   };
 
   const classes = classNames('c-btn', {
-    'c-btn--primary': primary,
-    'c-btn--destructive': destructive,
+    [`c-btn--${variant}`]: variant,
+    'c-btn--destructive': destructive && variant !== 'secondary',
```

Comment:
Should a destructive button be of a secondary variant? Shouldn't it always be a primary button?

Reply:
No, we already have a case where we use this lighter style for destructive button (see "Delete" next to drafts posts in Dashboard) and it does seem like we gonna have few more later on, in Admin side.